.github/workflows/pull-request-target.yml

```
@@ -0,0 +1,73 @@
+##############################################################################
+#
+# NOTE!
+#
+# Please read the README.md file in this directory that defines what should 
+# be placed in this file
+#
+##############################################################################
+##############################################################################
+
+name: PR Target Workflow
+on: 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)